kythe/go/extractors/config/smoke/smoke.go

```diff
@@ -163,16 +169,20 @@ func (g harness) filenamesFromRepo(repoURL string) (map[string]bool, error) {
 	return ret, err
 }
 
-func (g harness) filenamesFromExtraction(repoURL string) (map[string]bool, error) {
-	repoName := pathTail(repoURL)
+func (g harness) filenamesFromExtraction(repoURI string) (map[string]bool, error) {
+	repoName := pathTail(repoURI)
 	tmpOutDir, err := ioutil.TempDir("", repoName)
 	if err != nil {
-		return nil, fmt.Errorf("failed to create temp dir for repo %s: %v", repoURL, err)
+		return nil, fmt.Errorf("failed to create temp dir for repo %s: %v", repoURI, err)
 	}
 	defer os.RemoveAll(tmpOutDir)
 
-	err = g.extractor.ExtractRepo(repoURL, tmpOutDir, g.configPath)
 	ret := map[string]bool{}
+	err = g.extractor.ExtractRepo(config.Repo{
+		URI:        repoURI,
+		OutputPath: tmpOutDir,
+		ConfigPath: g.configPath,
+	})
 	if err != nil {
 		return ret, err
```

Comment:
Can this be more simply `return nil, err` given that `ret` has nothing in it? (A `nil` map behaves like an empty map for reads)